rpms.in.yaml

```diff
@@ -0,0 +1,29 @@
+contentOrigin:
```

Comment:
is the idea behind putting the rpm lock configuration in the root that all the containers in this repo and share the same base image and lock config while also getting the free updates from konflux?